pyatlan/client/credential.py

```diff
@@ -47,6 +47,12 @@ def creator(self, credential: Credential, test: bool = True) -> CredentialRespon
         :returns: A CredentialResponse instance.
         :raises ValidationError: If the provided `credential` is invalid.
         """
+
+        if not test and (
+            credential.username is not None or credential.password is not None
+        ):
+            raise ErrorCode.UNABLE_TO_CREATE_CREDENTIAL.exception_with_parameters()
```

Comment:
How about simplifying this to:

```suggestion
if not test and any((credential.username, credential.password)):
    raise ErrorCode.UNABLE_TO_CREATE_CREDENTIAL.exception_with_parameters()
```

Reply:
Done